Campus Density/Controllers/HomeFeedbackViewController.swift

```diff
@@ -0,0 +1,282 @@
+            prevButton.layer.borderWidth = 1
+            prevButton.layer.borderColor = UIColor.densityGreen.cgColor
+            prevButton.setTitleColor(.densityGreen, for: .normal)
+            prevButton.setTitle("Previous", for: .normal)
+            prevButton.titleLabel?.font = .fourteen
+            prevButton.isHidden = true
+            background.addSubview(prevButton)
+            prevButton.snp.makeConstraints { make in
+                make.height.equalTo(30)
+                make.width.equalTo(80)
+                make.bottom.left.equalToSuperview().inset(30)
+            }
+
+            thanksView = ThankYouQuestion()
+            thanksView.isHidden = true
+            background.addSubview(thanksView)
+            thanksView.snp.makeConstraints { make in
+                make.top.equalToSuperview().inset(30)
+                make.left.right.equalToSuperview().inset(20)
+                make.bottom.equalTo(nextButton.snp.top).offset(-20)
+            }
+
+        notAnswered = UILabel()
+        notAnswered.textColor = .black
+        notAnswered.textAlignment = .center
+        notAnswered.font = .ten
+        notAnswered.text = "Please answer this question before proceeding"
+        notAnswered.isHidden = true
+        background.addSubview(notAnswered)
+        notAnswered.snp.makeConstraints { make in
+            make.bottom.equalTo(nextButton.snp.top).offset(-20)
+            make.centerX.equalToSuperview()
+        }
+
+        }
+
+    func setupQuestions() {
+        for question in questions {
+            question.removeFromSuperview()
+        }
+
+        let recQuestion = RecommendQuestion()
+        recQuestion.delegate = self
+
+        let featuresQuestion = FeaturesQuestion()
+        featuresQuestion.delegate = self
+
+        let reviewQuestion = ReviewQuestion()
+        reviewQuestion.delegate = self
+
+        let otherCommentsQuestion = OtherCommentsQuestion()
+        otherCommentsQuestion.delegate = self
+
+        questions = [recQuestion, featuresQuestion, reviewQuestion, otherCommentsQuestion]
+        for question in questions {
+            background.addSubview(question)
+            question.snp.makeConstraints { make in
+                make.top.equalToSuperview().inset(30)
+                make.left.right.equalToSuperview().inset(30)
+                make.bottom.equalTo(nextButton.snp.top).offset(-30)
+            }
+            question.isHidden = true
+        }
+    }
+
+    func setupGestureRecognizers() {
+        let tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(dismissKeyboard))
+            tapRecognizer.cancelsTouchesInView = false
+            view.addGestureRecognizer(tapRecognizer)
+    }
+
+    func showWith() {
+        view.isHidden = false
+        navigationController?.navigationBar.barTintColor = UIColor(white: 1, alpha: 0)
+        feedback = FluxFeedback(likelytoRecommend: -1, usefulFeatures: [], likeFluxOverall: -1, comment: "")
+        }
```

Comment:
Dang swiftlint why didn't you catch this (more below too)

Reply:
lol fixed it manually